sentry-android-core/src/main/java/io/sentry/android/core/internal/gestures/AndroidComposeViewUtils.java

```diff
@@ -0,0 +1,102 @@
+package io.sentry.android.core.internal.gestures;
+
+import android.view.View;
+import androidx.compose.ui.layout.LayoutCoordinatesKt;
+import androidx.compose.ui.layout.ModifierInfo;
+import androidx.compose.ui.node.LayoutNode;
+import androidx.compose.ui.platform.AndroidComposeView;
+import androidx.compose.ui.semantics.SemanticsActions;
+import androidx.compose.ui.semantics.SemanticsConfiguration;
+import androidx.compose.ui.semantics.SemanticsModifier;
+import androidx.compose.ui.semantics.SemanticsProperties;
+import io.sentry.util.Objects;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@SuppressWarnings("KotlinInternalInJava")
+final class AndroidComposeViewUtils {
```

Comment:
@romtsn this is the secret sauce for Compose. Once we're happy with what we're doing here I suggest we switch to using reflection or add some safe guards around this.

Reply:
Ideally this lives in our compose lib as well - unfortunately only `AndroidComposeView` exposes the root layout node. Maybe it's fine to add a `compileOnly` dependency?